marginaleffects/sanity.py

```diff
@@ -262,6 +262,12 @@ def clean(k):
             hi = newdata[variable] + value / 2
             lo = newdata[variable] - value / 2
 
+        elif callable(value):
+            tmp = value(newdata[variable])
```

Comment:
Can you insert a check to make sure that `value` returns a data frame with two columns? If it doesn't, return an informative error.